test/unittest/compiler/unittest_realizer.cpp

```diff
@@ -163,10 +163,10 @@ TEST(SliceRealizer, slice_p) {
   std::vector<LayerRepresentation> after = {
     {"fully_connected", {"name=a1"}},
     {"fully_connected", {"name=b1", "input_layers=a1"}},
+    {"fully_connected", {"name=b2", "input_layers=a2"}},
```

Comment:
Could you add a negative test this patch is trying to solve as a regression?

Reply:
This change is just maintaining the orders of the node as defined in `before`.
This is the limitation of the `graphEqual` - https://github.com/nnstreamer/nntrainer/blob/988a6346d189b65c638fcefe7adf609f090ab700/test/unittest/compiler/compiler_test_util.cpp#L35

Reply:
Yes, not about this line. But a regression test which this patch resolves.

Reply:
done.